test/parse.js

```diff
@@ -530,4 +530,32 @@ describe('parse', function () {
       done()
     })
   })
+
+  it('parse richtext', function (done) {
+    var code = '<container> <richtext> <text if="a" repeat="list"> {{xxx}} </text> <image id="{{x}}" class="a {{x}} c" src="{{y}}" style="opacity: {{z}}"></image> </richtext> <container> <text if="a" repeat="list">{{xxx}}</text> <image id="{{x}}" class="a {{x}} c" src="{{y}}" style="opacity: {{z}}"></image> </container> </container>'
```

Comment:
`if` and `repeat` should be transformed into JavaScript statement I guess like:

```javascript
var self = this
self.list.filter(function ($value, $index) {
  return self.a
}).map(function ($value, $index) {
  return [
    { type: 'text', ...},
    { type: 'image', ...}
  ]
})
```

It's a little challenge I think.